src/data_gradients/feature_extractors/object_detection/bounding_boxes_area.py

```diff
@@ -131,7 +136,7 @@ def _compute_histogram(df: pd.DataFrame, transform_name: str) -> dict:
             split_data = df[df["split"] == split]
 
             dict_bincount[split] = {
-                "transform": transform_name,
+                "transform": name,
```

Comment:
Not sure about this. I like the idea of having a transform representing only the transformation, in your case it would also include info about "What is being transformed".
maybe we can keep `"transform": transform_name` but add a new key `"name": name`

For instance
```
{"transform": "sqrt", "name": "bbox_area"}
Or
{"transform": "sqrt", "name": "bbox_perimeter"}
```

Reply:
I don't think the transforms work, because now for example it's just a devision.